pkg/webhooks/cohort_webhook.go

```diff
@@ -75,6 +75,6 @@ func validateCohort(cohort *kueuealpha.Cohort) field.ErrorList {
 	var allErrs field.ErrorList
 
 	allErrs = append(allErrs, validateFairSharing(cohort.Spec.FairSharing, path.Child("fairSharing"))...)
-	allErrs = append(allErrs, validateResourceGroups(cohort.Spec.ResourceGroups, config, path.Child("resourceGroups"))...)
+	allErrs = append(allErrs, validateResourceGroups(cohort.Spec.ResourceGroups, config, path.Child("resourceGroups"), true)...)
```

Comment:
can we cover this message with a test case?

Reply:
I've added a test for cohort where error detail message is not ignored and compared to the expected message